src/Compilers/CSharp/Test/Emit/Emit/DynamicAnalysis/DynamicInstrumentationTests.cs

```diff
@@ -776,6 +831,78 @@ static int Lambda(int x, Func<int, int> l)
 True
 True
 True
+True
+";
+
+            CompileAndVerify(source + InstrumentationHelperSource, expectedOutput: expectedOutput, options: TestOptions.ReleaseExe);
+            CompileAndVerify(source + InstrumentationHelperSource, expectedOutput: expectedOutput, options: TestOptions.DebugExe);
+        }
+
+        [Fact]
+        public void MultipleFilesCoverage()
```

Comment:
I can of course add checking the IL, but I don't think that doing so would increase the robustness of the testing at all. I think that we would see "Source Document 0/1/2/3", because the fake indices are handed out on a zero-origin basis.